src/main/java/br/ufs/hu/prevsep/web/api/dto/form/sepse/PageableDoctorFormDTO.java

```diff
@@ -68,12 +73,17 @@ public void setDtCriacaoEnd(LocalDate dtCriacaoEnd) {
     @Override
     public Predicate getQueryPredicate(QFormularioSepseMedicoEntity qEntity) {
         BooleanBuilder filter = new BooleanBuilder();
+        BooleanBuilder subfilter = new BooleanBuilder();
         if (this.idFormulario != null)
             filter.and(qEntity.idFormulario.eq(this.idFormulario));
         if (this.crmMedico != null)
             filter.and(qEntity.crmMedico.eq(this.crmMedico));
         if (this.status != null)
-            filter.and(qEntity.status.eq(status.getValue()));
+            subfilter = new BooleanBuilder();
```

Comment:
A  condição deveria ser this.status != null && !this.status.empty(), pra só entrar aqui quando a lista não for vazia, evitando gerar null pointer dentro do próximo for.